templates/web/base/tokens/confirm_update.html

```diff
@@ -1,24 +1,24 @@
 [% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title => loc('Confirmation') %]
 
-<div class="confirmation-header">
-  <h1>[% loc('Your update has been confirmed!') %]</h1>
-
-<p class="confirmed">
 [% IF creator_fixed %]
-[%
-    tprintf(loc('Thank you &mdash; you can <a href="%s">view your updated problem</a> on the site.'),
-        c.uri_for( '/report', problem_id ) );
-%]
+  [% problem_url = c.uri_for('/report', problem_id) %]
 [% ELSE %]
-[%
-    tprintf( loc('You have successfully confirmed your update and you can now <a href="%s">view it on the site</a>.'),
-        c.uri_for( '/report', update.problem.id ) _ '#update_' _ update.id );
-%]
+  [% problem_url = c.uri_for( '/report', update.problem.id ) _ '#update_' _ update.id %]
 [% END %]
-</p>
+
+<div class="confirmation-header">
+
+  <h1><a href="[% problem_url %]">[% update.problem.title %]</a></h1>
+
+  <h2>[% loc('Thank you for updating this issue!') %]</h2>
```

Comment:
Sorry, I probably wasn't clear enough about `creator_fixed`. If a reporter leaves an update and marks the report as fixed, and hasn't answered this question before, then they are taken to `templates/web/base/questionnaire/creator_fixed.html` which asks them an immediate cut-down questionnaire (just asking if they've reported something before or not); when they fill this in, they then get this template. So they've already been told thanks for updating the issue when they were given the small questionnaire. Probably happens rarely enough that it doesn't really matter, but I thought I'd mention it in case you thought otherwise :)